Before/Office-Add-in-ASPNET-SSO-WebAPI/Scripts/Office/en-us/office_strings.debug.js

```diff
@@ -175,7 +175,7 @@ Strings.OfficeOM.L_GetSelectionNotSupported='The current selection is not suppor
 Strings.OfficeOM.L_EventHandlerAdditionFailed='Failed to add the event handler.';
 Strings.OfficeOM.L_DialogNavigateError='Dialog Navigation Error';
 Strings.OfficeOM.L_InvalidResourceUrl='Invalid application resource Url provided.';
-Strings.OfficeOM.L_InvalidOrTimedOutSessionMessage='Your Office Online session has expired or is invalid. To continue, refresh the page.';
+Strings.OfficeOM.L_InvalidOrTimedOutSessionMessage='Your Office on the web session has expired or is invalid. To continue, refresh the page.';
```

Comment:
"Office on the web session" doesn't sound right to me. Maybe we can do something like this instead? @Rick-Kirkham: Would this suggestion be accurate?
```suggestion
Strings.OfficeOM.L_InvalidOrTimedOutSessionMessage='Your Office browser session has expired or is invalid. To continue, refresh the page.';
```

Reply:
If this suggestion is accepted, do update the other instances.

Reply:
@ElizabethSamuel-MSFT  @lindalu-MSFT I think "browser session" is not quite right. Let's use a prepositional phrase:

"Your session with Office on the web has expired or is invalid."

Reply:
@Rick-Kirkham @lindalu-MSFT What about "Your session in Office on the web..."?

Reply:
@ElizabethSamuel-MSFT @lindalu-MSFT  That's fine with me.

Reply:
Changed to "Office browser session." Less wordy, more accurate.

Reply:
@lindalu-MSFT However, @Rick-Kirkham noted that Office browser session is not quite right...

Reply:
"Your session **with** Office on the web has expired or is invalid" isn't quite right either...

"The browser session running Office on the web has expired or is invalid" is clunky.

Reply:
As we discussed, maybe "Your Office session..." since the user should only see this in a browser context. @Rick-Kirkham?

Reply:
I can live with that.